scripts/misc/import_oldhgmd.pl

```diff
@@ -0,0 +1,668 @@
+  }
+
+  debug($config, "Sorting files based on the phenotype_feature_id column");
+  system("sort -k 1 -o ${TMP_DIR}${sorted_old_pheno_feat} ${TMP_DIR}${old_pheno_feat_file}");
+  system("sort -k 1 -o ${TMP_DIR}${sorted_old_pfa} ${TMP_DIR}${old_pheno_feat_attrib_file}");
+
+  debug($config, "Assigns new variation_ids and variation_feature_ids using the existing maximum variation_id and variation_feature_id");
+  manipulate_var_ids($dbh);
+
+  debug($config, "Manipulating Phenotype ids in each file");
+  manipulate_pheno_ids($dbh);
+
+  debug($config, "Sorting the new variation_feature file by seq regions. Test mode no insertion");
+  system("sort -k2,2n -k3,3n -k4,4n -o ${TMP_DIR}${sorted_new_vf} ${TMP_DIR}${new_var_feat}");
+
+
+  debug($config, "Sorting the new phenotype_feature file by seq regions. Test mode no insertion"); 
+  system("sort -k7,7n -k8,8n -k9,9n -o ${TMP_DIR}${sorted_new_pf} ${TMP_DIR}${new_pf_file}");
+}
+
+sub main { 
+  
+   my $old_dbname = $old_dbh->dbname;
+
+  debug($config, "Dumping HGMD data from tables variation and variation_feature from the $old_dbname");
+  dump_vdata_into_file($old_dbh);
+
+  debug($config, "Dumping old phenotype data with HGMD as source from $old_dbname");
+  dump_pheno($old_dbh);
+
+  debug($config, "Sorting files based on the variation_id column");
+  $TMP_DIR = $TMP_DIR . "/";
+  system("sort -k 1 -o ${TMP_DIR}${sorted_old_var} ${TMP_DIR}${old_var_file}");
+  system("sort -k 6 -o ${TMP_DIR}${sorted_old_var_feat} ${TMP_DIR}${old_vf_file}");
+  
+  if ($config->{transcript}) {
+    debug($config, "Sorting transcript variation files based on the variation_feature_id column");
+    system("sort -k 2 -o ${TMP_DIR}${sorted_old_tv_file} ${TMP_DIR}${old_tv_file}");
+    system("sort -k 2 -o ${TMP_DIR}${sorted_old_mtmp_file} ${TMP_DIR}${old_mtmp_file}");
+  }
+
+  debug($config, "Sorting files based on the phenotype_feature_id column");
+  system("sort -k 1 -o ${TMP_DIR}${sorted_old_pheno_feat} ${TMP_DIR}${old_pheno_feat_file}");
+  system("sort -k 1 -o ${TMP_DIR}${sorted_old_pfa} ${TMP_DIR}${old_pheno_feat_attrib_file}");
+
+  debug($config, "Assigns new variation_ids and variation_feature_ids using the maximum variation_id and variation_feature_id");
+  manipulate_var_ids($dbh);
+
+  debug($config, "Assigns new phenotype_feature id using the existing maximum phenotype_feature_id");
+  manipulate_pheno_ids($dbh);
+
+  debug($config, "Inserting HGMD data into variation table");
+  insert_into_variation_table($dbh, $new_var);
+
+  debug($config, "Sorting the new variation_feature file by seq regions");
+  system("sort -k2,2n -k3,3n -k4,4n -o ${TMP_DIR}${sorted_new_vf} ${TMP_DIR}${new_var_feat}");
+  debug($config, "Inserting HGMD data into variation_feature table");
+  insert_variation_feature($dbh, $sorted_new_vf);
+
+  if ($config->{transcript}) {
+    debug($config, "Inserting HGMD data into transcript_variation table");
+    insert_transcript_variation($dbh, $new_tv_file);
```

Comment:
The file here does not include the tmp dir `${TMP_DIR}` but when the file was created a tmp dir was used.